apps/game/README.md

```diff
@@ -15,50 +15,50 @@ $ iex -S mix
 1. Create a Game Server, providing a shortcode and player (only id required for now)
 
 ```elixir
-$ iex> Game.ServerSupervisor.start_game("ABCD", %{ id: "Gamer" })
+$ iex> GameApp.ServerSupervisor.start_game("ABCD", %GameApp.Player{ name: "fatal1ty", id: "Gamer" })
 ```
 
 2. Get a summary of the game state (shortcode required to locate correct game server process to message)
 
 ```elixir
-$ iex> Game.Server.summary("ABCD")
-
-%Game{
-  creator: %{id: "Gamer"},
-  funmaster: nil,
-  funmaster_order: [],
-  phase: :lobby,
-  players: %{"Gamer" => %{id: "Gamer"}},
-  round_number: nil,
-  rounds: [],
-  scores: %{},
-  shortcode: "ABCD",
-  winner: nil
+$ iex> GameApp.Server.summary("ABCD")
+
+%{
+creator: %GameApp.Player{id: "gamer", name: "fatal1ity"},
```

Comment:
Should this line also be indented? And I'm curious where the convention for having larger indents here comes from.

Reply:
Not sure, I just ran my editor's autoformat on it...but it looks like I don't have an `editorconfig` plugin set for vim. I'll fix that.